Update/docs faq ascat (#673)
Clinical-Genomics/BALSAMIC: docs/FAQs.rst

3. `bumpversion --verbose [major/minor/patch]`
4. `git push`
5. `git push --tags`

**References**
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**How to generate reference files for ascatNGS**

Detailed information is available at the [ascatNGS](https://github.com/cancerit/ascatNgs)

Briefly, ascatNGS needs gender loci file if gender information for the imput sample is not available. Second file is SnpGcCorrections.tsv, which is prepared from 1000 genome SNP panel.

> 1. Gender loci file:
>  - GRCh37d5_Y.loci contains the following contents:

Reviewer: ```suggestion
1. Gender loci file:

GRCh37d5_Y.loci contains the following contents:
```

```
Y       4546684
Y       2934912
Y       4550107
Y       4549638
```

> 2. GC correction file:
>  - First step is to download 1000 genome snp file and convert it from .vcf to .tsv. The detailed procedure to for this step is provided [here](https://github.com/cancerit/ascatNgs/wiki/Human-reference-files-from-1000-genomes-VCFs).

Reviewer: ```suggestion
2. GC correction file:

First step is to download 1000 genome snp file and convert it from .vcf to .tsv. The detailed procedure to for this step is provided [here](https://github.com/cancerit/ascatNgs/wiki/Human-reference-files-from-1000-genomes-VCFs).
```

```console
export TG_DATA=ftp://ftp.ensembl.org/pub/grch37/release-83/variation/vcf/homo_sapiens/1000GENOMES-phase_3.vcf.gz
```

followed by

``` console
curl -sSL $TG_DATA | zgrep -F 'E_Multiple_observations' | grep -F 'TSA=SNV' |\
perl -ane 'next if($F[0] !~ m/^\d+$/ && $F[0] !~ m/^[XY]$/); next if($F[0] eq $l_c && $F[1]-1000 < $l_p); $F[7]=~m/MAF=([^;]+)/; next if($1 < 0.05); printf "%s\t%s\t%d\n", $F[2],$F[0],$F[1]; $l_c=$F[0]; $l_p=$F[1];' \
> SnpPositions_GRCh37_1000g.tsv

```

--or--

```console
curl -sSL $TG_DATA | zgrep -F 'E_Multiple_observations' | grep -F 'TSA=SNV' |\
 perl -ane 'next if($F[0] !~ m/^\d+$/ && $F[0] !~ m/^[XY]$/); $F[7]=~m/MAF=([^;]+)/; next if($1 < 0.05); next if($F[0] eq $l_c && $F[1]-1000 < $l_p); printf "%s\t%s\t%d\n", $F[2],$F[0],$F[1]; $l_c=$F[0]; $l_p=$F[1];' \
> SnpPositions_GRCh37_1000g.tsv
```

>  - Second step is to use SnpPositions.tsv file and generate SnpGcCorrections.tsv file as described [here](https://github.com/cancerit/ascatNgs/wiki/Convert-SnpPositions.tsv-to-SnpGcCorrections.tsv).

Reviewer: ```suggestion
 Second step is to use SnpPositions.tsv file and generate SnpGcCorrections.tsv file as described [here](https://github.com/cancerit/ascatNgs/wiki/Convert-SnpPositions.tsv-to-SnpGcCorrections.tsv).
```

Author: @hassanfa I mistakenly added markdown codes. now fixed!

3. `bumpversion --verbose [major/minor/patch]`
4. `git push`
5. `git push --tags`

**References**
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**How to generate reference files for ascatNGS**

Detailed information is available at the ascatNGS_

.. _ascatNGS: https://github.com/cancerit/ascatNgs

> Briefly, ascatNGS needs gender loci file if gender information for the imput sample is not available. Second file is SnpGcCorrections.tsv, which is prepared from 1000 genome SNP panel.

Reviewer 2: ```suggestion
Briefly, ascatNGS needs a gender loci file if gender information for the input sample is not available. The second file is `SnpGcCorrections.tsv`, which is prepared from the 1000 genome SNP panel.
```

> 1. Gender loci file:

Reviewer 2: ```suggestion

1. **Gender loci file:**
```

  
GRCh37d5_Y.loci contains the following contents:

.. line-block::
    Y	4546684
    Y	2934912
    Y	4550107
    Y	4549638

> 2. GC correction file:

Reviewer 2: ```suggestion
2. **GC correction file:**
```

 
First step is to download 1000 genome snp file and convert it from .vcf to .tsv. The detailed procedure to for this step is provided here_

.. here_: https://github.com/cancerit/ascatNgs/wiki/Human-reference-files-from-1000-genomes-VCFs

.. code:: console

    export TG_DATA=ftp://ftp.ensembl.org/pub/grch37/release-83/variation/vcf/homo_sapiens/1000GENOMES-phase_3.vcf.gz

followed by:

.. code:: console

    curl -sSL $TG_DATA | zgrep -F 'E_Multiple_observations' | grep -F 'TSA=SNV' |\
    perl -ane 'next if($F[0] !~ m/^\d+$/ && $F[0] !~ m/^[XY]$/);\
    next if($F[0] eq $l_c && $F[1]-1000 < $l_p); $F[7]=~m/MAF=([^;]+)/;\
    next if($1 < 0.05); printf "%s\t%s\t%d\n", $F[2],$F[0],$F[1];\
    $l_c=$F[0]; $l_p=$F[1];' > SnpPositions_GRCh37_1000g.tsv

--or--

.. code:: console

    curl -sSL $TG_DATA | zgrep -F 'E_Multiple_observations' | grep -F 'TSA=SNV' |\
    perl -ane 'next if($F[0] !~ m/^\d+$/ && $F[0] !~ m/^[XY]$/); $F[7]=~m/MAF=([^;]+)/;\
    next if($1 < 0.05); next if($F[0] eq $l_c && $F[1]-1000 < $l_p);\
    printf "%s\t%s\t%d\n", $F[2],$F[0],$F[1]; $l_c=$F[0]; $l_p=$F[1];'\
    > SnpPositions_GRCh37_1000g.tsv

> Second step is to use SnpPositions.tsv file and generate SnpGcCorrections.tsv file as described here_

Reviewer 2: ```suggestion
Second step is to use `SnpPositions.tsv` file and generate `SnpGcCorrections.tsv` file as described here_
```
